networklibrary/src/main/java/com/eis/smsnetwork/SMSInvitation.java

```diff
@@ -0,0 +1,24 @@
+package com.eis.smsnetwork;
+
+import com.eis.communication.Peer;
+import com.eis.communication.network.Invitation;
+import com.eis.smslibrary.SMSPeer;
+
+/**
+ * Represents a received invitation to join a network.
+ */
+public class SMSInvitation implements Invitation<SMSPeer> {
+    SMSPeer inviter;
```

Comment:
Is it necessary for `inviter` to be package-private

Reply:
Im not sure. Definitely not public by looking at the constructor, but I think it might just be private. I'll carefully check with our current develop and keep you updated 👍

Reply:
Okay, I checked for that variable, it can be private without any problem, so I'll fix it now 👍